spec/fixtures/vcr_cassettes/Add_Interaction_To_Fern/Edit_Fern/can_add_an_interaction_to_a_fern.yml

```diff
@@ -0,0 +1,113 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: http://localhost:5000/api/v1/users/110920554030325122207/ferns/1
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Fern-Key:
+      - FErnCultForLyfe
```

Comment:
are we okay having this up on GH? or should this directory be in the `gitignore`